src/components/StoreMigrator/index.tsx

```diff
@@ -0,0 +1,77 @@
+import React, { useEffect, useState } from 'react'
+import { useDispatch } from 'react-redux'
+import { addressBookImport } from 'src/logic/addressBook/store/actions'
+import { logError, Errors } from 'src/logic/exceptions/CodedException'
+import { MigrationMessage } from 'src/routes/migration/container'
+import { MIGRATION_ADDRESS } from 'src/routes/routes'
+
+const MAINET_URL = 'http://localhost:3000'
+const networks = [
+  {
+    safeUrl: 'http://localhost:3001/#',
+  },
+  {
+    safeUrl: 'http://localhost:3002/#',
+  },
+]
+
+type MigrationMessageEvent = MessageEvent & {
+  data: MigrationMessage
+}
+
+const StoreMigrator: React.FC = () => {
+  const [currentNetwork, setCurrentNetwork] = useState(0)
+  const dispatch = useDispatch()
+  //let networks = getNetworks()
```

Comment:
Needs to be removed if not used.

Reply:
After the QA testing, we should use the getNetworks to get from where we're going to migrate